integration_testing/features/add-channel-to-starred- tab.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Add channel to *Starred* tab
+	User needs to be able to mark channels with a star to label them as favorite for easy access
+
+	Background: 
+		Given that I am signed in on the system

```

Comment:
`system` sounds too _generic_, let's switch all occurrences to `I am signed in to Studio`.